go/billing/tests/test_tasks.py

```diff
@@ -1286,7 +1287,8 @@ def test_credit_cutoff_email_sent(self):
         self.assertEqual(len(mail.outbox), 1)
         [email] = mail.outbox
 
-        self.assertEqual(email.recipients(), [self.django_user.email])
+        self.assertEqual(email.recipients(),
+                         [self.django_user.email, 'support@vumi.org'])
```

Comment:
`email.bcc` contains the list of bcc'ed addresses so we could add an assert like:
```python
self.assertEqual(email.bcc, ['support@vumi.org'])
```
in addition to the current assert.